File: Java/src/classes/CardioExercise.java
package classes;

public class CardioExercise extends Exercise {

    private double distance;

    public CardioExercise(String name, int duration, double distance) {
        super(name, duration);
        this.distance = distance;
    }

    @Override
    public double calcCalories() {
        return distance * 50;
    }

    @Override
    public String showInfo() {
        return "[Cardio] " + name + " - " + duration + "min - " + distance + "km";
    }
}

File: Python/classes/cardio_exercise.py
from classes.exercise import Exercise

class CardioExercise(Exercise):

    def __init__(self, name, duration, distance):
        super().__init__(name, duration)
        self.distance = distance
    
    def calc_calories(self):
        return self.distance * 50

    def show_info(self):
        return "[Cardio] " + self.name + " - " + str(self.duration) + "min - " + str(self.distance) + "km"